test/numerical/TestGemm.cpp

```diff
@@ -159,8 +159,9 @@ bool isOMGemmTheSameAsNaiveImplFor(const int I, const int J, const int K,
 int main(int argc, char *argv[]) {
   llvm::FileRemover remover(getSharedLibName(SHARED_LIB_BASE.str()));
 
+  setCompilerOption(OptionKind::CompilerOptLevel, "3");
   llvm::cl::ParseCommandLineOptions(
-      argc, argv, "TestGemm\n", nullptr, "TEST_ARGS");
+      argc, argv, "TestGEMM\n", nullptr, "TEST_ARGS");
```

Comment:
I think this ends up matching the filename in how it was capitalized for all the other tests, but no longer here.

Reply:
sure.